src/functions-and-arrays.js

```diff
@@ -29,8 +91,37 @@ const wordsUnique = [
   'bring'
 ];
 
+
+var uniquifyArray = function(array){
+  //var storedLength = array.length;
+  if (array.length === 0) {
+    return null;
+  }
+  for (i = 0; i < array.length; i++) {
```

Comment:
You can actually do this with a forEach, it would be easier and it would pass all the tests:

array.forEach(function (word) {
    if (!uniqueArray.includes(word)) {
        uniqueArray.push(word)
    };
})